--- Mail_Mime-1.3.1/usr/share/pear/Mail/mime.php	2006-01-24 19:26:17.000000000 +0200
+++ /tmp/mime.php	2006-01-24 19:23:43.000000000 +0200
@@ -590,7 +590,8 @@
         }
         $this->_headers = array_merge($headers, $this->_headers);
 
-        return $this->_encodeHeaders($this->_headers);
+        $tmp = $this->_encodeHeaders($this->_headers);
+        return $tmp;
     }
 
     /**
--- Mail_Mime-1.3.1/usr/share/pear/Mail/mime.php	2006-01-25 13:24:59.000000000 +0200
+++ /tmp/mime.php	2006-01-25 13:24:44.000000000 +0200
@@ -317,7 +317,8 @@
         if (is_object($obj)) {
             return $obj->addSubpart($text, $params);
         } else {
-            return new Mail_mimePart($text, $params);
+            $tmp = new Mail_mimePart($text, $params);
+            return $tmp;
         }
     }
 
--- Mail_Mime-1.3.1.orig/usr/share/pear/Mail/mime.php	2006-08-25 18:36:36.000000000 +0200
+++ Mail_Mime-1.3.1/usr/share/pear/Mail/mime.php	2006-08-25 18:37:44.000000000 +0200
@@ -354,7 +354,8 @@
     function &_addMixedPart()
     {
         $params['content_type'] = 'multipart/mixed';
-        return new Mail_mimePart('', $params);
+        $part = new Mail_mimePart('', $params);
+        return $part;
     }
 
     /**
@@ -431,7 +432,8 @@
         $params['encoding']     = $value['encoding'];
         $params['disposition']  = 'attachment';
         $params['dfilename']    = $value['name'];
-        $obj->addSubpart($value['body'], $params);
+        $part = $obj->addSubpart($value['body'], $params);
+        return $part;
     }
 
     /**
